requirements.txt

```diff
@@ -9,19 +9,20 @@ path.py
 Flask-Markdown
 Flask-WTF
 Flask-Mail
-Flask-WeasyPrint
 coloredlogs<=14.0
 query_phenomizer
 Flask-Babel
 livereload
 python-dateutil
 pymongo>=3.7,<4.0
 pathlib
-WeasyPrint==0.42.3
 pdfkit
 xlsxwriter
 click
 cryptography<3.4
+defusedxml
```

Comment:
These ones were probably part of Flask-WeasyPrint before

Reply:
Yes, requirements for it and WeasyPrint likely. Ok, good.